plugins/sonarqube/src/api/index.ts

```diff
@@ -105,6 +107,8 @@ export class SonarQubeApi {
       lastAnalysis: component.component.analysisDate,
       metrics,
       projectUrl: `${this.baseUrl}dashboard?id=${componentKey}`,
+      issuesUrl: `${this.baseUrl}project/issues?id=${componentKey}`,
+      componentMeasuresUrl: `${this.baseUrl}component_measures?id=${componentKey}`,
```

Comment:
How about something like this. Saves you an extra function in the `SonarQubeCard` and will be reusable in other cards easier.

```suggestion
      getIssueUrl: (type: string) => `${this.baseUrl}project/issues?id=${componentKey}&types=${type.toUpperCase()}&resolved=false`,
      getComponentMeasurementUrl: (metric: string) => `${this.baseUrl}component_measures?id=${componentKey}&metric=${metric.toLowerCase()}&resolved=false&view=list`,
    };
```